src/shared/IosAndroidSafeAreaView.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * The default react native SafeAreaView only works for iOS.
+ * This component works for iOS and Android to prevent View from overlapping
+ * with notifications bar or sensor area.
+ * Support for passing through props is limited and will probably have
+ * different effects on iOS/Android.
+ */
+import * as React from "react";
+import {
+  Text,
+  View,
+  SafeAreaView,
+  ViewProps,
+  Platform,
+  StatusBar
+} from "react-native";
+
+export const IosAndroidSafeAreaView: React.StatelessComponent<
```

Comment:
Can we just call it `SafeAreaView`? or `SaferAreaView` lol. Don't like the platform-specific naming.